Sprint-3/2-practice-tdd/get-ordinal-number.js

```diff
@@ -1,5 +1,39 @@
 function getOrdinalNumber(num) {
-  return "1st";
+  if (arguments.length !== 1) {
+    throw new Error("Function requires exactly one argument");
+  }
+  if (typeof num !== "number" || isNaN(num)) {
+    throw new Error("Input must be a number");
+  }
+  if (!Number.isFinite(num)){
+    throw new Error("Input must be a finite number");
+  }
+  if (!Number.isInteger(num) || num < 0) {
+    throw new Error("Input must be a non-negative integer");
+  }
```

Comment:
This is a good validation but I think this could be simplified because at the end your input should be a `Non-negative integer`. For example, `Number.isFinite()` and `isNan()` have some overlap, can you check in the MDN docs? 

I am leaving this as an optional comment, no action needs to be taken to complete this PR but I would like you think if this could be written simpler. You do not need to throw different error for each small case, you can have more general error for anything that is negative non-integer.